src/scala/microsoft-spark-2.4.x/src/main/scala/org/apache/spark/deploy/dotnet/DotnetRunner.scala

```diff
@@ -147,7 +147,7 @@ object DotnetRunner extends Logging {
       } else {
         // scalastyle:off println
         println("***********************************************************************")
```

Comment:
Can you update https://github.com/dotnet/spark/blob/master/src/scala/microsoft-spark-2.3.x/src/main/scala/org/apache/spark/deploy/dotnet/DotnetRunner.scala as well? That's causing the test failures.